### PR TITLE
GeekCorner.threema version 1.6.0.85

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22584)

### CODE REVIEW

File: manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GeekCorner.threema
+PackageVersion: 1.6.0.85
```
Comment (ImJoakim) [inline manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.installer.yaml:5]: image

```suggestion
PackageVersion: 1.6.0
```

File: manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GeekCorner.threema
+PackageVersion: 1.6.0.85
```
Comment (ImJoakim) [inline manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 1.6.0
```

File: manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: GeekCorner.threema
+PackageVersion: 1.6.0.85
```
Comment (ImJoakim) [inline manifests/g/GeekCorner/threema/1.6.0.85/GeekCorner.threema.yaml:5]: image

```suggestion
PackageVersion: 1.6.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/22584#pullrequestreview-717043877

**Review by ImJoakim — COMMENTED** [review]
Remember to also update the version folder as well 👍

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**